sql/src/test/java/io/crate/integrationtests/VersionHandlingIntegrationTest.java

```diff
@@ -203,4 +203,85 @@ public void testSelectWhereVersionWithPrimaryKey() throws Exception {
         expectedException.expectMessage("\"_version\" column is not valid in the WHERE clause of a SELECT statement");
         execute("select _version from test where col1 = 1 and _version = 50");
     }
+
+    @Test
+    public void testUpdateWhereVersionUsingWrongOperator() throws Exception {
+        execute("create table test (col1 integer primary key, col2 string)");
```

Comment:
can you move some of these into a WhereClauseValidator unit tests instead of running full blown integration tests for all different variants?